src/components/Navbar/Navbar.jsx

```diff
@@ -13,17 +17,17 @@ const Navbar = () => {
         </li>
         <li className="filter-item">
           <Link className="navbar-item" to="/">
-            All
+            All{allLength}
```

Comment:
A bit of design improvement 😁 
```suggestion
            All {allLength}
```

Reply:
Done. :))))